docs/ocean/design_docs/vert-lagrang-remap/vert-lagrang-remap.rst

```diff
@@ -0,0 +1,540 @@
+.. role:: red
+
+Vertical Lagrangian-remap method
+================================
+
+date: 2020/12/10
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+
+
+Summary
+-------
+
+We propose to implement the vertical Lagrangian-remap method to support the 
+choice of hybrid vertical coordinates, which may be required for evolving ice-
+shelf cavity geometries. It can be seen as an extension of the Arbitrary 
+Lagrangian-Eulerian method which is already implemented in MPAS-Ocean, a key 
+difference being that the solution of diasurface transport is coordinate-free. 
+This has the advantage of removing the vertical CFL constraint, which is of 
+particular concern for an evolving ice shelf geometry in terrain-following 
+coordinate schemes. This development may also reduce numerical diffusion in the
+vertical, but we do not require this result for this implementation to be 
+successful.
+
+Generalization of the vertical coordinate is not in the scope of this development. 
+Additionally, the infrastructure for specifying a target grid that differs from 
+the existing z-star and z-tilde options is reserved for future development.
+The minimum criteria for success are not degrading the accuracy of the solution 
+or the performance by more than [an acceptable threshold] for several basic test 
+cases. 
+*[not quite sure how to write the success statement if not "meets requirements"]*
```

Comment:
@dengwirda @xylar Thoughts on criteria for success?

Reply:
I think what @vanroekel discussed below is correct.  It will be some combination of "expert judgment" and somewhat more quantitative criteria such as Salil's test to determine if the code modifications change the climate outside of ensemble variability.